server/channels/api4/apitestlib.go

```diff
@@ -102,6 +102,17 @@ func setupTestHelper(dbStore store.Store, searchEngine *searchengine.Broker, ent
 	*memoryConfig.AnnouncementSettings.AdminNoticesEnabled = false
 	*memoryConfig.AnnouncementSettings.UserNoticesEnabled = false
 	*memoryConfig.PluginSettings.AutomaticPrepackagedPlugins = false
+	// Enabling Redis with Postgres.
+	if *memoryConfig.SqlSettings.DriverName == model.DatabaseDriverPostgres {
```

Comment:
>  but assuming Redis is running because Postgres is might prove brittle.

Ah no no, the point here was that to run the Redis enabled tests only for Postgres, but to run LRU tests for MySQL. This is similar to the GraphQL issue, where you can run the entire test suite once with GraphQL enabled, and then disabled.

The challenge here is whether to run Redis tests for ALL cases (that would mean zero coverage for LRU), or only for some cases. And if it's the latter, then how to split them. I had another idea to randomly enable the Redis config so that some tests randomly run with Redis, but it feels like that would lead to even more confusion.

So, from what I am getting, you are suggesting something like `make test-server-redis`, and then run everything with Redis?

Reply:
Oh, I see! I have no objections to the "split coverage" approach (though you could also just flip a coin each run for even more fun and coverage), but I guess the takeaway is that `redis` should now be a default Docker service. Should we update `config.mk` to add `redis` to this baseline?

Reply:
Ah yes definitely, let me do that.